lib/split/experiment_catalog.rb

```diff
@@ -48,5 +50,8 @@ def self.normalize_experiment(metric_descriptor)
     end
     private_class_method :normalize_experiment
 
+    def self.clear_cache
+      @cache = {}
+    end
```

Comment:
seems like `ExperimentCatalog#clear_cache` is not being called anywhere 🤔

Reply:
Good catch! That is left over code from before refactoring out a more generic `Split::Cache`.  Will remove.